core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -324,6 +324,12 @@ private synchronized void initServer() throws IOException {
                 .getMs(PropertyKey.MASTER_EMBEDDED_JOURNAL_TRANSPORT_REQUEST_TIMEOUT_MS),
             TimeUnit.MILLISECONDS));
 
+    RaftServerConfigKeys.RetryCache.setExpiryTime(properties,
+        TimeDuration.valueOf(
+            ServerConfiguration.global()
```

Comment:
Can this not be `ServerConfiguration.getMs()`?

Reply:
Good to know. Updated.